lib/containers/docker.pm

```diff
@@ -62,7 +63,13 @@ sub test_built_img() {
     assert_script_run("mkdir templates");
     assert_script_run("cd templates");
     assert_script_run("wget --quiet " . data_url('containers/index.html'));
-    assert_script_run("docker run -dit -p 8888:5000 -v ~/templates:\/usr/src/app/templates myapp https://openqa.suse.de//api/v1/jobs/${id}");
+    if (is_sle()) {
```

Comment:
you can also do 1 liner 
`$path = is_sle() ? "openqa.suse.de" : "openqa.opensuse.org";`

However, I'm not sure I like this solution, we are still hardcoding 2 URLs and this makes things not really portable (e.g. I can't execute this test on my local or any other environment). I've seen other tests providing the url as an external variable, e.g. [boot_from_pxe.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/boot/boot_from_pxe.pm#L62).

Maybe you can do something like:

```
my $path = is_sle() ? "openqa.suse.de" : "openqa.opensuse.org";`
$path = get_var('OPENQA_URL', $path);
```
or similar...

Reply:
Thank you Jose for the feedback. The automatic variable OPENQA_URL, will return the localhost and not the public IP (as also other variables, like host_ip and OPENQA_HOSTNAME)
I haven't found a variable to return the public IP (eg when I execute locally, I need the IP of the machine with the openqa installed).
In any case, I will modify a little bit the test, but keep the logic that checks the network outside the container as it is.

Reply:
Can you point me where you see it returning localhost?

I've seen some jobs in OSD and it returns `"OPENQA_URL": "openqa.suse.de",`
https://openqa.suse.de/tests/4285682/file/vars.json

Reply:
For the openqa.suse.de it works, but when somebody wants to clone the test and run it locally, as I mentioned before, the OPENQA_URL will not return the public ip of the local openqa machine and the test will fail.